src/bot/modules/stats.ts

```diff
@@ -88,28 +86,26 @@ export default class Stats {
         })
       ]);
 
-      const [suggestions_approvals, corrections_approvals] = partition(
-        approvals,
+      const [suggestions, corrections] = partition(
+        [...approvals, ...disapprovals],
         (approval) => approval.proposal.type === ProposalType.SUGGESTION
       );
-      const [suggestions_disapprovals, corrections_disapprovals] = partition(
-        disapprovals,
-        (disapproval) => disapproval.proposal.type === ProposalType.SUGGESTION
-      );
-      const ratio = (approvals.length / disapprovals.length) * 100;
+
+      const ratio_approvals = (approvals.length / [...approvals, ...disapprovals].length) * 100;
+      const ratio_disapprovals = (disapprovals.length / [...approvals, ...disapprovals].length) * 100;
       fields.push({
         name: 'Décisions de Parrain :',
         value: stripIndents`
-          Décisions totales: **${approvals.length + disapprovals.length}**
-          Suggestions: **${approveEmoji} ${suggestions_approvals.length} ${disapproveEmoji} ${
-          suggestions_disapprovals.length
-        }**
-          Corrections: **${approveEmoji} ${corrections_approvals.length} ${disapproveEmoji}  ${
-          corrections_disapprovals.length
-        }**
-          Ratio: **${ratio ? `${ratio}` : 0}%**
+          Décisions totales: **${[...approvals, ...disapprovals].length}**
```

Comment:
Pas de raison de faire un double spread operator
```suggestion
          Décisions totales: **${totalDecisions}**
```